optimum/intel/openvino/modeling.py

```diff
@@ -464,17 +461,33 @@ def _from_transformers(
             force_download=force_download,
             trust_remote_code=trust_remote_code,
             ov_config=ov_config,
-            library_name="transformers",
+            library_name=library_name,
         )
 
         config.save_pretrained(save_dir_path)
-        return cls._from_pretrained(
+        model = cls._from_pretrained(
             model_id=save_dir_path,
             config=config,
             load_in_8bit=load_in_8bit,
             quantization_config=quantization_config,
             **kwargs,
         )
+        if len(model.model.outputs) == 2:  # Sentence Transormers output
```

Comment:
I think it is also good to check names as model can have 2 output due to different reasons